docs/zh/latest/pubsub.md

```diff
@@ -0,0 +1,248 @@
+---
+title: 发布订阅框架
+keywords:
+  - APISIX
+  - Pub-Sub
+description: This document contains information about the Apache APISIX pub-sub framework.
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 摘要
+
+发布订阅是一种消息范式，
+
+- 消息生产者不直接将消息发送给消息消费者，而是由特定的代理进行中转
+- 代理会将生产者发送的消息缓存下来，之后主动推送至订阅的消费者或由消费者拉取
+
+在系统架构中通常使用这种模式进行系统解耦，或是处理大流量场景。
+
+在 Apache APISIX 中，最常用的场景是用于处理服务器至客户端的南北向流量，如果可以结合发布订阅系统，我们可以实现更为强大的功能，例如在线文档实时协作、在线游戏等。
+
+## 架构
+
+![pub-sub architecture](../../assets/images/pubsub-architecture.svg)
+
+当前，Apache APISIX 支持以 WebSocket 与客户端通信，客户端可以是任何支持 WebSocket 的程序，以 Protocol Buffer 作为序列化机制，查看[协议定义](../../../apisix/pubsub.proto)。
+
+## 当前支持的消息系统
+
+- [Aapche Kafka](pubsub/kafka.md)
```

Comment:
I think this is WIP, all right? pls update your doc

Reply:
and I think you can submit the English version only, we can submit the Chinese version later

small PR is easier review

Reply:
removed both of them